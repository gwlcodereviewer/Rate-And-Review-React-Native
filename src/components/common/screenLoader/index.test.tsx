import React from 'react';
import renderer from 'react-test-renderer';
import ScreenLoader from '.';

let wrapper: renderer.ReactTestRenderer;

describe('render correctly', () => {
  wrapper = renderer.create(<ScreenLoader />);

  it('render correctly screenLoader view', () => {
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
