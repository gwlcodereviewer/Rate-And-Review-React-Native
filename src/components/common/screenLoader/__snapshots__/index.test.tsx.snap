// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly render correctly screenLoader view 1`] = `
<View
  color="transparent"
  style={
    Array [
      Object {
        "backgroundColor": "transparent",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
      },
    ]
  }
>
  <ActivityIndicator
    color="#1778F2"
    size="large"
  />
</View>
`;
